stable/python/Makefile

```diff
@@ -131,9 +120,17 @@ post-python-verify:
 	echo $$logs | grep -q "event-id.*"
 
 post-python-custom-port:
-	kubeless function deploy post-python-custom-port --runtime python2.7 --handler hellowithdata.handler --from-file examples/hellowithdata.py --port 8081
+	kubeless function deploy post-python-custom-port --runtime python3.6 --handler hellowithdata.handler --from-file examples/hellowithdata.py --port 8081
 
 post-python-custom-port-verify:
 	kubectl rollout status deployment/post-python-custom-port && sleep 2
 	kubectl get svc post-python-custom-port -o yaml | grep 'targetPort: 8081'
 	kubeless function call post-python-custom-port --data '{"it-s": "alive"}'|egrep "it.*alive"
+
+python-preload:
+	cd examples && zip preload.zip preload.py preloadhelper.py
+	kubeless function deploy python-preload --runtime python3.6 --handler preload.handler --from-file examples/preload.zip --timeout 3
+
+python-preload-verify:
+	kubectl rollout status deployment/python-preload && sleep 60
```

Comment:
I've deliberately added two sleep's of 10 seconds each in the example file - `examples/preload.py`- to simulate an operation that takes long to complete and show the performance benefit of `forkserver` compared to `spawn` (each request takes few milliseconds for `forkserver` vs 20 seconds for `spawn`).

However, before we start serving requests, `preload.py` gets loaded twice.
1. It gets loaded first time when we do `mod = importlib.import_module(os.getenv('MOD_NAME'))`
2. It gets loaded a second time inside the forkserver when the forkserver is started - `d = ctx.Process(target=preload)`

So, the total delay before we can start serving requests is 2 * (10 + 10) = 40 seconds. 
I just added 20 seconds on top of that as a safety net.

I can reduce the sleep duration in `preload.py` to 4 seconds (2 + 2 for each of the ML models) and reduce the sleep in this Makefile to 2 * (2 + 2) = 8 + 2 (safety net) = 10 seconds.

Ideally we would need to add a `readinessProbe` so that `kubectl rollout status` will tell us exactly when the pod is ready to serve requests.

Reply:
> I can reduce the sleep duration in `preload.py` to 4 seconds (2 + 2 for each of the ML models) and reduce the sleep in this Makefile to 2 * (2 + 2) = 8 + 2 (safety net) = 10 seconds.

Yes, that'd be better.

Reply:
I have made the change.